schedule-tracking/src/main/java/org/motechproject/scheduletracking/domain/json/ScheduleRecord.java

```diff
@@ -6,12 +6,18 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * <code>ScheduleRecord</code> is used to deserialize schedule from json format.
```

Comment:
it is not used to "deserialize"; it just keeps the deserialized form of a schedule